sycl/test-e2e/KernelAndProgram/target_register_alloc_mode.cpp

```diff
@@ -2,12 +2,14 @@
 
 // RUN: %{build} -ftarget-register-alloc-mode=pvc:auto -o %t_with.out
 // RUN: %{build} -o %t_without.out
+// RUN: %{build} -ftarget-register-alloc-mode=pvc:default -o %t_default.out
 
-// RUN: env SYCL_PI_TRACE=-1 %{run} %t_with.out 2>&1 | FileCheck --check-prefix=CHECK-WITH %s
-// RUN: env SYCL_PI_TRACE=-1 %{run} %t_without.out 2>&1 | FileCheck --implicit-check-not=-ze-intel-enable-auto-large-GRF-mode %s
+// RUN: env SYCL_PI_TRACE=-1 %{run} %t_with.out 2>&1 | FileCheck --check-prefix=CHECK-OPT %s
+// RUN: env SYCL_PI_TRACE=-1 %{run} %t_without.out 2>&1 | FileCheck %if system-windows %{ --implicit-check-not=-ze-intel-enable-auto-large-GRF-mode %} %else %{ --check-prefix=CHECK-OPT %} %s
+// RUN: env SYCL_PI_TRACE=-1 %{run} %t_default.out 2>&1 | FileCheck --implicit-check-not=-ze-intel-enable-auto-large-GRF-mode %s
 
-// CHECK-WITH: ---> piProgramBuild(
-// CHECK-WITH: -ze-intel-enable-auto-large-GRF-mode
+// CHECK-OPT: ---> piProgramBuild(
```

Comment:
could you please clarify what is the difference between check-with and check-opt? I haven't find any related stuff in FileCheck impl for both of them.

Reply:
sorry, it's just a custom prefix i'm using, it's not a special option inside FileCheck like CHECK-COUNT or something. you can see i set check-prefix in FileCheck calls at the top

Reply:
yes, I noticed your change of command lines. So it is just a custom prefix that is checked like **must-to-have** in output content, right?

Reply:
yes it's the exact same behavior is `CHECK`, it is must have or it will fail

Reply:
thanks for the review!